docs/pages/reference/user-types.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: User Types
+description: Describes the different types of Teleport users and their properties.
+keywords: [user,idp,sso]
+tocDepth: 3
+---
+
+This guide explains the different kinds of users in Teleport, how they are
+created, and their properties.
+
+## Local Users
+
+Local users are created directly in Teleport. They are not coming from an
+external system like an identity provider.
+
+Local users can be created via the CLI (`tctl users add`), by applying a user
+resource manifest (`tctl create -f user.yaml`) or via the web UI. Their roles
+and traits can be modified directly in Teleport.
+
+Those users can connect to Teleport directly via username/password and/or
+additional authentication factors such as webauthn physical tokens, passkeys or
+One Time Passwords.
+
+Local user login can be disabled via `cluster_auth_preference` or `teleport.yaml`.
+Disabling local authentication is required for [FIPS/FedRAMP compliance
+](../access-controls/compliance-frameworks/fedramp.mdx).
+
+### Special case: Bots
+
+Machine ID provides machines with an identity that can authenticate to the
+Teleport cluster. This identity is known as a bot. Bots are represented in
+Teleport by a user and a role resource and can be created via the
+`tctl bots add` command.
+
+Bots are not logging in like human users with a password, MFA or SSO. They join
+the cluster as Teleport services using [a join method](./join-methods.mdx). They
+can still join even if local auth is disabled. 
+
+<Admonition type="note" title="See also">
+The [Machine ID introduction](../machine-id/introduction.mdx).
+</Admonition>
+
+## SSO users
+
+Those users are imported in Teleport from an external identity provider.
+Teleport contains a user resource representing those users, but it's only a
+representation of a user of a remote system.
```

Comment:
I would add some more concrete details as to how this user resource differs from the one created for local users. Is the TTL different? Also, what about the user resource changes because it's a representation of a user of a remote system?

Reply:
There is no difference between SSO users and local users currently. Temporary users are a subgenre of sso users and always have an expiry. SSO Sync-ed users have no expiry. Local users might or might not have an expiry.